app/data_providers/slate_providers/new_tab_slate_provider.py

```diff
@@ -104,10 +104,11 @@ async def rank_corpus_items(self, items: List[CorpusItemModel], *args, **kwargs)
             default_alpha_prior=188,  # beta * P99 German NewTab CTR for 2023-03-28 to 2023-04-05 (1.5%)
             default_beta_prior=12500)  # 0.5% of median German NewTab item impressions for 2023-03-28 to 2023-04-05.
 
-        # 2. Secondary sort order is publisher diversity.
-        items = spread_publishers(items, spread_distance=PUBLISHER_SPREAD_DISTANCE)
-
-        # 1. Primary sort order is recency. Sort is stable, so it will preserve existing order within a scheduled date.
+        # 2. Secondary sort order is recency.
```

Comment:
I would be concerned this may negatively impact CTR.  My understanding is that there is a time decay applied somewhere in the feed machine such that most items drop after a day.  Depending on how many items you are loading here, I might suggest recency third and TS second.  Additionally, I recall there is some impression capping on the firefox side, so I think the recency concerns may be more limited.

Reply:
This was exactly the feedback I was looking for. TS is not a stable sort, because all values it sorts on are distinct, so any sort that would happen before it would not have any effect.

A penalty approach would be totally in our power to implement. Let's have the conversation!

Reply:
Captured this in [DIS-686](https://getpocket.atlassian.net/browse/DIS-686) so it won't get lost.